docs/book/installation/sylius_plus_installation.rst

```diff
@@ -291,9 +291,19 @@ Installing Sylius Plus as a plugin to a Sylius application
         use InventorySourceAwareTrait;
     }
 
-**7.** Add wkhtmltopdf binary for Invoicing purposes.
+**7.** Install wkhtmltopdf binary for Invoicing and Refund purposes.
 
-If you do not have the ``wkhtmltopdf`` binary, download it `here <https://wkhtmltopdf.org/downloads.html>`_.
+Default configuration assumes enabled PDF file generator. If you don't want to use that feature change your app configuration:
+
+.. code-block:: yaml
+
+    # config/packages/sylius_plus.yaml
+    sylius_plus:
```

Comment:
It would be nice to add an additional note for disabling PDF generation also for Invoicing and Refund plugins, because this one parameter for Plus wouldn't be enough for not using `wkhtmltopdf` binary